delivery_carrier_category/views/delivery_carrier_category.xml

```diff
@@ -67,7 +67,7 @@
         <field name="action" ref="delivery_carrier_category_act_window"/>
         <field name="sequence" eval="16"/>
     </record>
-    <record model="ir.ui.menu" id="delivery_carrier_category_menu">
+    <record model="ir.ui.menu" id="delivery_carrier_category_sale_menu">
```

Comment:
this item seems to be a full duplicate of the item line 64. should be better to remove it ? don't you think.

Reply:
Sure?
I chage it because the parent view is different : 
<field name="parent_id" ref="delivery.menu_delivery"/>
https://github.com/OCA/delivery-carrier/blob/7fe4de0e3fd811a1c5544279fe2d7808f0f1fb5d/delivery_carrier_category/views/delivery_carrier_category.xml#L66
<field name="parent_id" ref="delivery.sale_menu_delivery"/>
https://github.com/OCA/delivery-carrier/blob/7fe4de0e3fd811a1c5544279fe2d7808f0f1fb5d/delivery_carrier_category/views/delivery_carrier_category.xml#L62